app/src/main/java/net/frju/flym/ui/main/MainActivity.kt

```diff
@@ -288,6 +315,17 @@ class MainActivity : AppCompatActivity(), MainNavigator, AnkoLogger {
         }
     }
 
+    override fun onStart() {
+        super.onStart()
+
+        if (!hasHandeledOnStartIntent) {
```

Comment:
instead of handling that in onStart with an hasHandleledOnStartIntent boolean, why not doing it in onCreate ? I believe we can also change the intent via setIntent() to avoid doing the stuff again if the activity is recreated for some reason

Reply:
You're right it's better in onCreate, it's changed. You mean changing the intent to a "blank" one? I think if the activity is recreated it will be another intent anyway (to show the activity).